python/gradbench/pytorch/run.py

```diff
@@ -18,6 +20,41 @@ def run(params):
     return {"output": func.unwrap(ret), "nanoseconds": {"evaluate": end - start}}
 
 
+def adroit_path(module: str) -> Path:
+    folder = Path(__file__).parent / "tmp/adroit"
+    path = folder / f"{module}.adroit"
+    path.parent.mkdir(parents=True, exist_ok=True)
+    path = path.resolve()
+    if path.is_relative_to(folder):
+        return path
+    else:
+        raise ValueError(f"resolved path {path} is not relative to {folder}")
+
+
+def define(module, source):
+    try:  # File already exists
+        import_module(module)
+    except:  # Try to translate to create file
+        src = adroit_path(module)
+        src.write_text(source)
+        adroit = "usr/local/bin/adroit"
```

Comment:
Nice work on this!

One request: currently this only works in the Docker image because the cwd is set to `/`, but when running locally via Poetry, it breaks because this path isn't absolute. You actually don't need to specify the full path at all though, because the `ADD` command in the Dockerfile puts it on the `PATH`, so instead of specifying the full path you can just say `adroit`:

```suggestion
        adroit = "adroit"
```